modules/imgproc/misc/python/pyopencv_imgproc.hpp

```diff
@@ -0,0 +1,18 @@
+#ifdef HAVE_OPENCV_IMGPROC
```

Comment:
Since #8497 is already merged, the whole `pyopencv_imgproc.hpp` file is no longer needed.. Thanks @alalek